backend/app/adapter/sqldb/user_short_link_integration_test.go

```diff
@@ -114,6 +114,137 @@ func TestListShortLinkSql_FindAliasesByUser(t *testing.T) {
 	}
 }
 
+func TestListShortLinkSql_FindAliasByUser(t *testing.T) {
+	now := mustParseTime(t, "2019-05-01T08:02:16Z")
+	user := entity.User{
+		ID:             "test",
+		Name:           "mockedUser",
+		Email:          "test@example.com",
+		LastSignedInAt: &now,
+		CreatedAt:      &now,
+		UpdatedAt:      &now,
+	}
+
+	testCases := []struct {
+		name               string
+		userTableRows      []userTableRow
+		shortLinkTableRows []shortLinkTableRow
+		relationTableRows  []userShortLinkTableRow
+		alias              string
+		user               entity.User
+		hasErr             bool
```

Comment:
hasErr is false in each test case, can you make a test when this is true or remove the field hasErr from the struct?

Reply:
I will be removing this in one of the smaller PRs I'm creating :pray: ty for the feedback